lib/order.rb

```diff
@@ -0,0 +1,70 @@
+require_relative './customer'
+require 'csv'
+
+class Order
+  attr_reader :id, :products, :customer, :fulfillment_status
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = customer
+    @fulfillment_status = fulfillment_status
+    valid_statuses = [:paid, :processing, :pending, :shipped, :complete]
+    unless valid_statuses.include?(fulfillment_status)
+      raise ArgumentError, "Not a valid fulfillment status."
```

Comment:
It's helpful for debugging to include the invalid argument in your messages:

```suggestion
      raise ArgumentError, "Not a valid fulfillment status: #{fulfillment_status}"
```